examples/converters.py

```diff
@@ -0,0 +1,78 @@
+import discord
+from discord.ext import commands
+
+# We will need the members intents at the minimum to use the Member converter.
+intents = discord.Intents.all()
+
+# Construct the Bot instance.
+bot = commands.Bot("!", intents=intents)
+
+
+@bot.command()
+async def userinfo(ctx: commands.Context, user: discord.User):
+    # In the command signature about, you can see that the `user`
+    # parameter is typehinted to `discord.User`. This means that
+    # during command invocation we will attempt to convert
+    # the value passed as `user` to a `discord.User` instance.
+
+    # If the conversion is successful we will have a User instance
+    # and can do the following:
+    return await ctx.send(user.name)
+
+@userinfo.error
+async def userinfo_error(ctx: commands.Context, error: Exception):
+    # if the conversion above fails for any reason, it will raise `commands.errors.BadArgument`
+    # so we handle this in this error handler
+    if isinstance(error, commands.BadArgument):
+        return await ctx.send("Failed to convert the argument for `user` to `discord.User`.")
+
+# Manual use of converters will follow
+
+@bot.command()
+async def channel_or_member(ctx: commands.Context, argument: int):
```

Comment:
converters take strings not int so the typehint should not be `int`

Reply:
This is intending to use the `int` converter. This is explained in the comments just below.

Reply:
this will typeerror from memberconverter.convert not accepting ints